frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -1634,7 +1634,21 @@ window.AppVersionView = countlyView.extend({
             this.dtable = $('.d-table').dataTable($.extend({}, $.fn.dataTable.defaults, {
                 "aaData": appVersionData.chartData,
                 "aoColumns": [
-                    { "mData": "app_versions", "sTitle": jQuery.i18n.map["app-versions.table.app-version"] },
+                    {
+                        "mData": "app_versions",
+                        "sTitle": jQuery.i18n.map["app-versions.table.app-version"],
+                        "mRender": function(data, type) {
+                            if (type === "sort") {
+                                var numbers = data.split(".");
+                                data = "";
+                                for (var i in numbers) {
+                                    data += numbers[i];
+                                }
+                                parseInt(data);
```

Comment:
Firstly parseInt data should be used somewhere
Secondly, it will incorrectly sort app_versions 2.3.5 and 2.35
we need something more sophisticated
Maybe something like this:
https://github.com/Countly/countly-server/blob/master/bin/scripts/checking_versions.js#L63-L96
Or this:
https://github.com/Countly/countly-server/blob/master/api/utils/common.js#L1653-L1708